client/scripts/views/modals/new_topic_modal.ts

```diff
@@ -72,23 +106,24 @@ const NewTopicModal: m.Component<{
               const disallowedCharMatches = text.match(/["<>%{}|\\/^`]/g);
               if (disallowedCharMatches) {
                 return [
-                  TextInputStatus.Error,
-                  `The ${pluralizeWithoutNumberPrefix(disallowedCharMatches.length, 'char')} 
-                  ${disallowedCharMatches.join(', ')} are not permitted`
+                  ValidationStatus.Failure,
```

Comment:
I wonder if there's a way to abstract needing to import ValidationStatus whenever we use the inputs, like having an optional field in `CWTextInput` for a "validation script" that returns a boolean, and then the CWTextInput, having received the boolean, shows one of two string inputs for "success" + "failure", both of which are also optional attrs to `CWTextInput`. this can also be pushed back against for future refactoring/component design

Reply:
Yeah, I figured there was a better way to approach it but I applaud @gdjohnson for trying. Let's talk more about it soon.